src/subscriptions.js

```diff
@@ -0,0 +1,84 @@
+// Manages web push subscriptions
```

Comment:
note, the reason we are using a new local storage key for this is the `store` is basically predicated on the idea of a schema with known keys for configs, etc, but this has arbitrary keys (hub ids)